modules/perception/obstacle/camera/transformer/flat_camera_transformer.cc

```diff
@@ -25,14 +25,17 @@ bool FlatCameraTransformer::Transform(
     std::vector<std::shared_ptr<VisualObject>> *objects) {
   if (!objects) return false;
 
+  // Center in Camera Space and Derived Center in Ego Car Space
+  std::vector<std::pair<Eigen::Vector3f, Eigen::Vector3f>> centers;
```

Comment:
I still don't understand the meaning of the center. Can you explain it in detail?
Thank you!